test/runner/tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist = True
-minversion = 2.5.0
+minversion = 1.4.2
```

Comment:
This will pass CI since we're not using the `--tox` option there, but it won't work right.

We need [changes made in version 2.5.0 of tox](https://tox.readthedocs.io/en/latest/changelog.html#v2-5-0-2016-11-16), specifically:

> slightly backward incompatible: fix #310: the {posargs} substitution now properly preserves the tox command line positional arguments. Positional arguments with spaces are now properly handled. NOTE: if your tox invocation previously used extra quoting for positional arguments to work around #310, you need to remove the quoting. Example: tox – “‘some string’” # has to now be written simply as tox – “some string” thanks holger krekel. You can set minversion = 2.5.0 in the [tox] section of tox.ini to make sure people using your tox.ini use the correct version.